src/include/daos_types.h

```diff
@@ -197,8 +196,8 @@ enum {
  * It is put here because it's almost used by everyone.
  */
 typedef struct {
-	uint64_t	lo;
-	uint64_t	hi;
+	uint64_t	lo;	/**< reserved for DAOS */
+	uint64_t	hi;	/**< user provided */
```

Comment:
Don't think it is true. DAOS uses only the high 32 bits of daos_obj_id_t::hi

Reply:
Removed.  I was paraphrasing the existing comment on L191 but really just wanted to check that the doxygen annotations were being picked up correctly.